stable/clamav/templates/daemonset.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.daemonset.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "clamav.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ template "clamav.name" . }}
+    helm.sh/chart: {{ template "clamav.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "clamav.name" . }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ template "clamav.name" . }}
+    spec:
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        effect: NoSchedule
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        resources:
+          limits:
```

Comment:
Again, this limits should be commented on the values files, and users can uncomment if they whant them here.